man/mkbundle.1

```diff
@@ -21,9 +21,34 @@ default only the assemblies specified in the command line will be
 included in the bundle.  To automatically include all of the
 dependencies referenced, use the "--deps" command line option.
 .PP
-Use \fImkbundle\FP when you want the startup runtime to load the 1.0
-profile, and use \fImkbundle2\fP when you want the startup runtime to load
-the 2.0 profile.
+There are two modes of operation, the default one uses the
+C compiler to create a bundle and requires a complete C and Mono SDK
+to produced executables.   The simple mode (enabled when using the
+"--simple") command line option does not require this, and also allows
+for cross compilation.
+.PP
+For example, to create a bundle for hello world, use the following
+command:
+.nf
+	$ mkbundle -o hello --simple hello.exe
+.fi
+.PP
+The simple version allows for cross-compiling, this requires a Mono
+runtime to be installed in the ~/.mono/targets/TARGET/mono to be
+available.   You can use the "--local-targets" to list all available
+targets, and the "--cross" argument to specify the target, like this:
+.nf
+	$ mkbundle --local-targets	
+	Available targets:
+		default	- Current System Mono
+		4.4.0-macosx-x86
+		4.4.0-debian-8-arm64
+	$ mkbundle --cross 4.4.0-debian-8-powerpc hello.exe -o hello-debian
```

Comment:
`4.4.0-debian-8-powerpc` is not listed in `available targets`.

Btw. what are the plans regarding Mono versions here? Do we push all releases there? If yes, should this use the 4part version like 4.4.0.168 instead?